transformato/state.py

```diff
@@ -1032,13 +1032,14 @@ def _init_base_dir(self):
     def _write_toppar_str(self, output_file_base):
 
         from transformato.system import parameter_files
-
         base = "../../toppar"
         toppar_format = ""
         for i in parameter_files:
```

Comment:
It seems like the `parameter_files` variable is always the one for the last structure, for `asfe` that's fine but in case of `rsfe/rbfe` this means that we are always using the parameters necessary for structure 2!